include/circt/Dialect/HW/Passes.td

```diff
@@ -58,4 +58,14 @@ def VerifyInnerRefNamespace : Pass<"hw-verify-irn"> {
   let constructor =  "circt::hw::createVerifyInnerRefNamespacePass()";
 }
 
+def LowerHWtoBTOR2 : Pass<"hw-lower-to-botr2", "mlir::ModuleOp"> {
+  let summary = "Lowers the hw dialect to a btor2 format";
+  let constructor = "circt::hw::createLowerHWtoBTOR2Pass()";
+  let description = [{
+    Creates a btor2 model file from the hw dialect. This requires tracking
+    the btor-id to mlir-name mappings and available bitvec sort sizes in order
+    to express the hw model in a btor form. 
+  }];
+}
```

Comment:
Ok I'll take a look at that, I simply did it this way because I didn't know of other ways to do it, but if a "translation pass" concept exists, then this should definitely be that ^^"